src/App.js

```diff
@@ -1,19 +1,53 @@
+import { render } from '@testing-library/react';
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import {useState} from 'react'; 
+
+
+
+const INITIAL_ENTRIES = chatMessages
 
 const App = () => {
+  const initialCopy = INITIAL_ENTRIES.map(chatEntry => {
+    return {...chatEntry};
+  })
+
+  
+
+  const [chatLog] = useState(initialCopy);
+
+  const updateLikeCount = (hearts)=> {
+    console.log(hearts)
+    if(hearts) {
+    setLikesCount(likesCount +1)}
+    else {
+    setLikesCount(likesCount -1)
+    }
+
+
+
+  }
+  
+  
+  const [likesCount, setLikesCount] = useState(0)
+  
+
+  render() ;
```

Comment:
This line should not be in the App component and is causing a warning to be thrown.